doc/DataPlaneCodeGeneration/DeveloperDrivenEvolution.md

```diff
@@ -205,3 +210,76 @@ namespace Azure.AI.MetricsAdvisor
     }
 }
 ```
+
+### Implement LRO method
+#### Case: With RequestContent but no return value
+**Generated code before (Generated/DeviceManagementClient.cs):**
+```C#
+namespace Azure.IoT.DeviceUpdate
+{
+    public partial class DeviceManagementClient
+    {
+        public virtual Operation ImportDevices(WaitUntil waitUntil, string action, RequestContent content, RequestContext context = null)
+        {
+            using var scope = ClientDiagnostics.CreateScope("DeviceManagementClient.ImportDevices");
+        }
+    }
+}
+```
+**Improve the LRO method ({Other manual folder}/DeviceManagementClient.cs):**
+```C#
+namespace Azure.IoT.DeviceUpdate
+{
+    public partial class DeviceManagementClient
+    {
+        // The return type is still Operation because no response value
+        public virtual Operation ImportDevices(WaitUntil waitUntil, string action, ImportType type, CancellationToken cancellationToken = default)
```

Comment:
What is 'ImportType', a model?  Do i need to implement a function called 'ImportType.ToRequestContent'?